packages/sw-lib/test/sw/claim-clients-false.js

```diff
@@ -1,9 +1,8 @@
 importScripts('/node_modules/mocha/mocha.js');
 importScripts('/node_modules/chai/chai.js');
-importScripts('/node_modules/sinon/pkg/sinon.js');
+importScripts('/node_modules/sinon/pkg/sinon-no-sourcemaps.js');
```

Comment:
Oh, that.

I was seeing weirdness with the previous `sinon.js` was pulled in via `importScripts` in my
 local Chrome installation—it would fail to evaluate properly, and cause service worker registration to fail. I wasn't able to figure out what specifically was causing it to choke, but using the `-no-sourcemaps.js` version worked fine, and is equivalent.

I can still reproduce it, so maybe I'll take some time and investigate further and then file a bug against https://github.com/sinonjs/sinon